src/test/java/core/service.impl/ReaderImplTest.java

```diff
@@ -0,0 +1,46 @@
+package core.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.Reader;
+import core.basesyntax.service.impl.ReaderImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class ReaderImplTest {
+    private static final String FILE_PATHNAME = "src/test/resources/fileone.csv";
+    private static final String EMPTY_FILE = "src/test/resources/emptyFile.csv";
+    private static final String NOT_EXISTING_FILE = "src/test/resources/noFile.csv";
+
+    private void fillExpectedList(List<String> expectedList) {
```

Comment:
Private ones (methods, classes) should be after public ones.
Use `get...` methods - see comments to ParserImplTest